components/tools/OmeroPy/src/omero/plugins/render.py

```diff
@@ -28,7 +28,7 @@
 from omero.model import Image
 from omero.model import Plate
 from omero.model import Screen
-from omero.util import pydict_text_readers
+from omero.util import pydict_text_io
```

Comment:
This breaks https://github.com/openmicroscopy/infrastructure/pull/61/files#diff-bf4435d4dde93c8ad0a4566ee4315a52R14 -- to be corrected in a follow-up PR.